apisix/plugins/ai-proxy/schema.lua

```diff
@@ -188,11 +188,16 @@ _M.ai_proxy_multi_schema = {
             default = { algorithm = "roundrobin" }
         },
         instances = ai_instance_schema,
+        fallback_strategy = {
+            type = "string",
+            enum = { "instance_health_and_rate_limiting" },
+            default = "instance_health_and_rate_limiting",
+        },
         timeout = {
             type = "integer",
             minimum = 1,
             maximum = 60000,
-            default = 3000,
+            default = 30000,
```

Comment:
do we need to update the related documents?

Reply:
fixed